[2025-03-28T05:37:09.701+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-28T05:37:09.720+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: split_web_scraping_v01.scrape_and_store manual__2025-03-28T05:37:07.809707+00:00 [queued]>
[2025-03-28T05:37:09.729+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: split_web_scraping_v01.scrape_and_store manual__2025-03-28T05:37:07.809707+00:00 [queued]>
[2025-03-28T05:37:09.730+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-28T05:37:09.747+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): scrape_and_store> on 2025-03-28 05:37:07.809707+00:00
[2025-03-28T05:37:09.756+0000] {standard_task_runner.py:72} INFO - Started process 260 to run task
[2025-03-28T05:37:09.761+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'split_web_scraping_v01', 'scrape_and_store', 'manual__2025-03-28T05:37:07.809707+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/split_web_scraping.py', '--cfg-path', '/tmp/tmp1npyfp81']
[2025-03-28T05:37:09.765+0000] {standard_task_runner.py:105} INFO - Job 85: Subtask scrape_and_store
[2025-03-28T05:37:09.838+0000] {task_command.py:467} INFO - Running <TaskInstance: split_web_scraping_v01.scrape_and_store manual__2025-03-28T05:37:07.809707+00:00 [running]> on host ce237b7c39e9
[2025-03-28T05:37:09.953+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='waiyong' AIRFLOW_CTX_DAG_ID='split_web_scraping_v01' AIRFLOW_CTX_TASK_ID='scrape_and_store' AIRFLOW_CTX_EXECUTION_DATE='2025-03-28T05:37:07.809707+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-28T05:37:07.809707+00:00'
[2025-03-28T05:37:09.957+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-28T05:37:10.012+0000] {logging_mixin.py:190} INFO - Processing batch 1: pages 1 to 10
[2025-03-28T05:37:10.013+0000] {logging_mixin.py:190} INFO - Scraping page 1...
[2025-03-28T05:37:10.040+0000] {logging_mixin.py:190} INFO - Error processing batch 1: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146e860>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.056+0000] {logging_mixin.py:190} INFO - Processing batch 2: pages 11 to 20
[2025-03-28T05:37:10.058+0000] {logging_mixin.py:190} INFO - Scraping page 11...
[2025-03-28T05:37:10.063+0000] {logging_mixin.py:190} INFO - Error processing batch 2: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=11 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146dae0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.082+0000] {logging_mixin.py:190} INFO - Processing batch 3: pages 21 to 30
[2025-03-28T05:37:10.083+0000] {logging_mixin.py:190} INFO - Scraping page 21...
[2025-03-28T05:37:10.092+0000] {logging_mixin.py:190} INFO - Error processing batch 3: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=21 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14526e0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.106+0000] {logging_mixin.py:190} INFO - Processing batch 4: pages 31 to 40
[2025-03-28T05:37:10.108+0000] {logging_mixin.py:190} INFO - Scraping page 31...
[2025-03-28T05:37:10.114+0000] {logging_mixin.py:190} INFO - Error processing batch 4: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=31 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451570>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.128+0000] {logging_mixin.py:190} INFO - Processing batch 5: pages 41 to 50
[2025-03-28T05:37:10.129+0000] {logging_mixin.py:190} INFO - Scraping page 41...
[2025-03-28T05:37:10.134+0000] {logging_mixin.py:190} INFO - Error processing batch 5: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=41 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146f9d0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.150+0000] {logging_mixin.py:190} INFO - Processing batch 6: pages 51 to 60
[2025-03-28T05:37:10.152+0000] {logging_mixin.py:190} INFO - Scraping page 51...
[2025-03-28T05:37:10.160+0000] {logging_mixin.py:190} INFO - Error processing batch 6: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=51 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452530>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.173+0000] {logging_mixin.py:190} INFO - Processing batch 7: pages 61 to 70
[2025-03-28T05:37:10.175+0000] {logging_mixin.py:190} INFO - Scraping page 61...
[2025-03-28T05:37:10.180+0000] {logging_mixin.py:190} INFO - Error processing batch 7: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=61 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452ad0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.196+0000] {logging_mixin.py:190} INFO - Processing batch 8: pages 71 to 80
[2025-03-28T05:37:10.197+0000] {logging_mixin.py:190} INFO - Scraping page 71...
[2025-03-28T05:37:10.204+0000] {logging_mixin.py:190} INFO - Error processing batch 8: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=71 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146ca00>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.221+0000] {logging_mixin.py:190} INFO - Processing batch 9: pages 81 to 90
[2025-03-28T05:37:10.223+0000] {logging_mixin.py:190} INFO - Scraping page 81...
[2025-03-28T05:37:10.229+0000] {logging_mixin.py:190} INFO - Error processing batch 9: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=81 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14504f0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.243+0000] {logging_mixin.py:190} INFO - Processing batch 10: pages 91 to 100
[2025-03-28T05:37:10.244+0000] {logging_mixin.py:190} INFO - Scraping page 91...
[2025-03-28T05:37:10.250+0000] {logging_mixin.py:190} INFO - Error processing batch 10: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=91 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14521d0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.263+0000] {logging_mixin.py:190} INFO - Processing batch 11: pages 101 to 110
[2025-03-28T05:37:10.265+0000] {logging_mixin.py:190} INFO - Scraping page 101...
[2025-03-28T05:37:10.272+0000] {logging_mixin.py:190} INFO - Error processing batch 11: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=101 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14503a0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.297+0000] {logging_mixin.py:190} INFO - Processing batch 12: pages 111 to 120
[2025-03-28T05:37:10.300+0000] {logging_mixin.py:190} INFO - Scraping page 111...
[2025-03-28T05:37:10.310+0000] {logging_mixin.py:190} INFO - Error processing batch 12: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=111 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146cfa0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.329+0000] {logging_mixin.py:190} INFO - Processing batch 13: pages 121 to 130
[2025-03-28T05:37:10.331+0000] {logging_mixin.py:190} INFO - Scraping page 121...
[2025-03-28T05:37:10.338+0000] {logging_mixin.py:190} INFO - Error processing batch 13: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=121 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452a10>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.355+0000] {logging_mixin.py:190} INFO - Processing batch 14: pages 131 to 140
[2025-03-28T05:37:10.356+0000] {logging_mixin.py:190} INFO - Scraping page 131...
[2025-03-28T05:37:10.364+0000] {logging_mixin.py:190} INFO - Error processing batch 14: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=131 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452cb0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.380+0000] {logging_mixin.py:190} INFO - Processing batch 15: pages 141 to 150
[2025-03-28T05:37:10.382+0000] {logging_mixin.py:190} INFO - Scraping page 141...
[2025-03-28T05:37:10.388+0000] {logging_mixin.py:190} INFO - Error processing batch 15: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=141 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451c60>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.409+0000] {logging_mixin.py:190} INFO - Processing batch 16: pages 151 to 160
[2025-03-28T05:37:10.410+0000] {logging_mixin.py:190} INFO - Scraping page 151...
[2025-03-28T05:37:10.416+0000] {logging_mixin.py:190} INFO - Error processing batch 16: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=151 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146d900>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.435+0000] {logging_mixin.py:190} INFO - Processing batch 17: pages 161 to 170
[2025-03-28T05:37:10.436+0000] {logging_mixin.py:190} INFO - Scraping page 161...
[2025-03-28T05:37:10.442+0000] {logging_mixin.py:190} INFO - Error processing batch 17: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=161 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450370>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.458+0000] {logging_mixin.py:190} INFO - Processing batch 18: pages 171 to 180
[2025-03-28T05:37:10.460+0000] {logging_mixin.py:190} INFO - Scraping page 171...
[2025-03-28T05:37:10.465+0000] {logging_mixin.py:190} INFO - Error processing batch 18: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=171 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452c80>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.478+0000] {logging_mixin.py:190} INFO - Processing batch 19: pages 181 to 190
[2025-03-28T05:37:10.479+0000] {logging_mixin.py:190} INFO - Scraping page 181...
[2025-03-28T05:37:10.483+0000] {logging_mixin.py:190} INFO - Error processing batch 19: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=181 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14506a0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.498+0000] {logging_mixin.py:190} INFO - Processing batch 20: pages 191 to 200
[2025-03-28T05:37:10.499+0000] {logging_mixin.py:190} INFO - Scraping page 191...
[2025-03-28T05:37:10.504+0000] {logging_mixin.py:190} INFO - Error processing batch 20: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=191 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146d990>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.517+0000] {logging_mixin.py:190} INFO - Processing batch 21: pages 201 to 210
[2025-03-28T05:37:10.518+0000] {logging_mixin.py:190} INFO - Scraping page 201...
[2025-03-28T05:37:10.524+0000] {logging_mixin.py:190} INFO - Error processing batch 21: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=201 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452e00>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.542+0000] {logging_mixin.py:190} INFO - Processing batch 22: pages 211 to 220
[2025-03-28T05:37:10.543+0000] {logging_mixin.py:190} INFO - Scraping page 211...
[2025-03-28T05:37:10.549+0000] {logging_mixin.py:190} INFO - Error processing batch 22: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=211 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451ae0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.566+0000] {logging_mixin.py:190} INFO - Processing batch 23: pages 221 to 230
[2025-03-28T05:37:10.567+0000] {logging_mixin.py:190} INFO - Scraping page 221...
[2025-03-28T05:37:10.571+0000] {logging_mixin.py:190} INFO - Error processing batch 23: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=221 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1453220>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.583+0000] {logging_mixin.py:190} INFO - Processing batch 24: pages 231 to 240
[2025-03-28T05:37:10.584+0000] {logging_mixin.py:190} INFO - Scraping page 231...
[2025-03-28T05:37:10.588+0000] {logging_mixin.py:190} INFO - Error processing batch 24: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=231 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146e410>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.600+0000] {logging_mixin.py:190} INFO - Processing batch 25: pages 241 to 250
[2025-03-28T05:37:10.601+0000] {logging_mixin.py:190} INFO - Scraping page 241...
[2025-03-28T05:37:10.605+0000] {logging_mixin.py:190} INFO - Error processing batch 25: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=241 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1453220>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.620+0000] {logging_mixin.py:190} INFO - Processing batch 26: pages 251 to 260
[2025-03-28T05:37:10.621+0000] {logging_mixin.py:190} INFO - Scraping page 251...
[2025-03-28T05:37:10.627+0000] {logging_mixin.py:190} INFO - Error processing batch 26: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=251 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452710>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.640+0000] {logging_mixin.py:190} INFO - Processing batch 27: pages 261 to 270
[2025-03-28T05:37:10.643+0000] {logging_mixin.py:190} INFO - Scraping page 261...
[2025-03-28T05:37:10.647+0000] {logging_mixin.py:190} INFO - Error processing batch 27: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=261 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14508e0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.662+0000] {logging_mixin.py:190} INFO - Processing batch 28: pages 271 to 280
[2025-03-28T05:37:10.663+0000] {logging_mixin.py:190} INFO - Scraping page 271...
[2025-03-28T05:37:10.668+0000] {logging_mixin.py:190} INFO - Error processing batch 28: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=271 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146d900>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.681+0000] {logging_mixin.py:190} INFO - Processing batch 29: pages 281 to 290
[2025-03-28T05:37:10.682+0000] {logging_mixin.py:190} INFO - Scraping page 281...
[2025-03-28T05:37:10.687+0000] {logging_mixin.py:190} INFO - Error processing batch 29: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=281 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14506a0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.700+0000] {logging_mixin.py:190} INFO - Processing batch 30: pages 291 to 300
[2025-03-28T05:37:10.701+0000] {logging_mixin.py:190} INFO - Scraping page 291...
[2025-03-28T05:37:10.705+0000] {logging_mixin.py:190} INFO - Error processing batch 30: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=291 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450640>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.719+0000] {logging_mixin.py:190} INFO - Processing batch 31: pages 301 to 310
[2025-03-28T05:37:10.721+0000] {logging_mixin.py:190} INFO - Scraping page 301...
[2025-03-28T05:37:10.726+0000] {logging_mixin.py:190} INFO - Error processing batch 31: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=301 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14504f0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.740+0000] {logging_mixin.py:190} INFO - Processing batch 32: pages 311 to 320
[2025-03-28T05:37:10.742+0000] {logging_mixin.py:190} INFO - Scraping page 311...
[2025-03-28T05:37:10.748+0000] {logging_mixin.py:190} INFO - Error processing batch 32: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=311 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146e230>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.762+0000] {logging_mixin.py:190} INFO - Processing batch 33: pages 321 to 330
[2025-03-28T05:37:10.763+0000] {logging_mixin.py:190} INFO - Scraping page 321...
[2025-03-28T05:37:10.769+0000] {logging_mixin.py:190} INFO - Error processing batch 33: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=321 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450af0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.786+0000] {logging_mixin.py:190} INFO - Processing batch 34: pages 331 to 340
[2025-03-28T05:37:10.787+0000] {logging_mixin.py:190} INFO - Scraping page 331...
[2025-03-28T05:37:10.793+0000] {logging_mixin.py:190} INFO - Error processing batch 34: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=331 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451120>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.809+0000] {logging_mixin.py:190} INFO - Processing batch 35: pages 341 to 350
[2025-03-28T05:37:10.811+0000] {logging_mixin.py:190} INFO - Scraping page 341...
[2025-03-28T05:37:10.817+0000] {logging_mixin.py:190} INFO - Error processing batch 35: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=341 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451480>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.833+0000] {logging_mixin.py:190} INFO - Processing batch 36: pages 351 to 360
[2025-03-28T05:37:10.834+0000] {logging_mixin.py:190} INFO - Scraping page 351...
[2025-03-28T05:37:10.839+0000] {logging_mixin.py:190} INFO - Error processing batch 36: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=351 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146d090>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.854+0000] {logging_mixin.py:190} INFO - Processing batch 37: pages 361 to 370
[2025-03-28T05:37:10.855+0000] {logging_mixin.py:190} INFO - Scraping page 361...
[2025-03-28T05:37:10.861+0000] {logging_mixin.py:190} INFO - Error processing batch 37: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=361 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452710>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.876+0000] {logging_mixin.py:190} INFO - Processing batch 38: pages 371 to 380
[2025-03-28T05:37:10.877+0000] {logging_mixin.py:190} INFO - Scraping page 371...
[2025-03-28T05:37:10.882+0000] {logging_mixin.py:190} INFO - Error processing batch 38: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=371 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14525f0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.896+0000] {logging_mixin.py:190} INFO - Processing batch 39: pages 381 to 390
[2025-03-28T05:37:10.897+0000] {logging_mixin.py:190} INFO - Scraping page 381...
[2025-03-28T05:37:10.902+0000] {logging_mixin.py:190} INFO - Error processing batch 39: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=381 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452140>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.917+0000] {logging_mixin.py:190} INFO - Processing batch 40: pages 391 to 400
[2025-03-28T05:37:10.918+0000] {logging_mixin.py:190} INFO - Scraping page 391...
[2025-03-28T05:37:10.923+0000] {logging_mixin.py:190} INFO - Error processing batch 40: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=391 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146ef80>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.937+0000] {logging_mixin.py:190} INFO - Processing batch 41: pages 401 to 410
[2025-03-28T05:37:10.938+0000] {logging_mixin.py:190} INFO - Scraping page 401...
[2025-03-28T05:37:10.944+0000] {logging_mixin.py:190} INFO - Error processing batch 41: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=401 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450640>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.958+0000] {logging_mixin.py:190} INFO - Processing batch 42: pages 411 to 420
[2025-03-28T05:37:10.959+0000] {logging_mixin.py:190} INFO - Scraping page 411...
[2025-03-28T05:37:10.964+0000] {logging_mixin.py:190} INFO - Error processing batch 42: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=411 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450f70>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.978+0000] {logging_mixin.py:190} INFO - Processing batch 43: pages 421 to 430
[2025-03-28T05:37:10.979+0000] {logging_mixin.py:190} INFO - Scraping page 421...
[2025-03-28T05:37:10.984+0000] {logging_mixin.py:190} INFO - Error processing batch 43: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=421 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451bd0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:10.999+0000] {logging_mixin.py:190} INFO - Processing batch 44: pages 431 to 440
[2025-03-28T05:37:11.001+0000] {logging_mixin.py:190} INFO - Scraping page 431...
[2025-03-28T05:37:11.005+0000] {logging_mixin.py:190} INFO - Error processing batch 44: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=431 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146df60>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.019+0000] {logging_mixin.py:190} INFO - Processing batch 45: pages 441 to 450
[2025-03-28T05:37:11.020+0000] {logging_mixin.py:190} INFO - Scraping page 441...
[2025-03-28T05:37:11.026+0000] {logging_mixin.py:190} INFO - Error processing batch 45: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=441 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452710>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.042+0000] {logging_mixin.py:190} INFO - Processing batch 46: pages 451 to 460
[2025-03-28T05:37:11.043+0000] {logging_mixin.py:190} INFO - Scraping page 451...
[2025-03-28T05:37:11.048+0000] {logging_mixin.py:190} INFO - Error processing batch 46: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=451 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14500a0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.063+0000] {logging_mixin.py:190} INFO - Processing batch 47: pages 461 to 470
[2025-03-28T05:37:11.064+0000] {logging_mixin.py:190} INFO - Scraping page 461...
[2025-03-28T05:37:11.069+0000] {logging_mixin.py:190} INFO - Error processing batch 47: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=461 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450070>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.084+0000] {logging_mixin.py:190} INFO - Processing batch 48: pages 471 to 480
[2025-03-28T05:37:11.085+0000] {logging_mixin.py:190} INFO - Scraping page 471...
[2025-03-28T05:37:11.090+0000] {logging_mixin.py:190} INFO - Error processing batch 48: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=471 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146e9b0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.104+0000] {logging_mixin.py:190} INFO - Processing batch 49: pages 481 to 490
[2025-03-28T05:37:11.105+0000] {logging_mixin.py:190} INFO - Scraping page 481...
[2025-03-28T05:37:11.112+0000] {logging_mixin.py:190} INFO - Error processing batch 49: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=481 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450850>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.128+0000] {logging_mixin.py:190} INFO - Processing batch 50: pages 491 to 500
[2025-03-28T05:37:11.129+0000] {logging_mixin.py:190} INFO - Scraping page 491...
[2025-03-28T05:37:11.135+0000] {logging_mixin.py:190} INFO - Error processing batch 50: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=491 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14500a0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.151+0000] {logging_mixin.py:190} INFO - Processing batch 51: pages 501 to 510
[2025-03-28T05:37:11.152+0000] {logging_mixin.py:190} INFO - Scraping page 501...
[2025-03-28T05:37:11.158+0000] {logging_mixin.py:190} INFO - Error processing batch 51: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=501 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450ca0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.175+0000] {logging_mixin.py:190} INFO - Processing batch 52: pages 511 to 520
[2025-03-28T05:37:11.179+0000] {logging_mixin.py:190} INFO - Scraping page 511...
[2025-03-28T05:37:11.185+0000] {logging_mixin.py:190} INFO - Error processing batch 52: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=511 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146e3e0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.209+0000] {logging_mixin.py:190} INFO - Processing batch 53: pages 521 to 530
[2025-03-28T05:37:11.210+0000] {logging_mixin.py:190} INFO - Scraping page 521...
[2025-03-28T05:37:11.218+0000] {logging_mixin.py:190} INFO - Error processing batch 53: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=521 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452f80>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.239+0000] {logging_mixin.py:190} INFO - Processing batch 54: pages 531 to 540
[2025-03-28T05:37:11.240+0000] {logging_mixin.py:190} INFO - Scraping page 531...
[2025-03-28T05:37:11.248+0000] {logging_mixin.py:190} INFO - Error processing batch 54: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=531 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450220>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.271+0000] {logging_mixin.py:190} INFO - Processing batch 55: pages 541 to 550
[2025-03-28T05:37:11.272+0000] {logging_mixin.py:190} INFO - Scraping page 541...
[2025-03-28T05:37:11.282+0000] {logging_mixin.py:190} INFO - Error processing batch 55: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=541 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451c60>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.300+0000] {logging_mixin.py:190} INFO - Processing batch 56: pages 551 to 560
[2025-03-28T05:37:11.301+0000] {logging_mixin.py:190} INFO - Scraping page 551...
[2025-03-28T05:37:11.307+0000] {logging_mixin.py:190} INFO - Error processing batch 56: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=551 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146e950>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.326+0000] {logging_mixin.py:190} INFO - Processing batch 57: pages 561 to 570
[2025-03-28T05:37:11.327+0000] {logging_mixin.py:190} INFO - Scraping page 561...
[2025-03-28T05:37:11.333+0000] {logging_mixin.py:190} INFO - Error processing batch 57: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=561 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452020>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.354+0000] {logging_mixin.py:190} INFO - Processing batch 58: pages 571 to 580
[2025-03-28T05:37:11.355+0000] {logging_mixin.py:190} INFO - Scraping page 571...
[2025-03-28T05:37:11.361+0000] {logging_mixin.py:190} INFO - Error processing batch 58: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=571 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451ab0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.375+0000] {logging_mixin.py:190} INFO - Processing batch 59: pages 581 to 590
[2025-03-28T05:37:11.376+0000] {logging_mixin.py:190} INFO - Scraping page 581...
[2025-03-28T05:37:11.380+0000] {logging_mixin.py:190} INFO - Error processing batch 59: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=581 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450340>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.393+0000] {logging_mixin.py:190} INFO - Processing batch 60: pages 591 to 600
[2025-03-28T05:37:11.394+0000] {logging_mixin.py:190} INFO - Scraping page 591...
[2025-03-28T05:37:11.398+0000] {logging_mixin.py:190} INFO - Error processing batch 60: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=591 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146fa90>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.411+0000] {logging_mixin.py:190} INFO - Processing batch 61: pages 601 to 610
[2025-03-28T05:37:11.412+0000] {logging_mixin.py:190} INFO - Scraping page 601...
[2025-03-28T05:37:11.416+0000] {logging_mixin.py:190} INFO - Error processing batch 61: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=601 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1453160>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.429+0000] {logging_mixin.py:190} INFO - Processing batch 62: pages 611 to 620
[2025-03-28T05:37:11.430+0000] {logging_mixin.py:190} INFO - Scraping page 611...
[2025-03-28T05:37:11.435+0000] {logging_mixin.py:190} INFO - Error processing batch 62: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=611 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452290>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.448+0000] {logging_mixin.py:190} INFO - Processing batch 63: pages 621 to 630
[2025-03-28T05:37:11.449+0000] {logging_mixin.py:190} INFO - Scraping page 621...
[2025-03-28T05:37:11.455+0000] {logging_mixin.py:190} INFO - Error processing batch 63: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=621 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451030>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.470+0000] {logging_mixin.py:190} INFO - Processing batch 64: pages 631 to 640
[2025-03-28T05:37:11.471+0000] {logging_mixin.py:190} INFO - Scraping page 631...
[2025-03-28T05:37:11.477+0000] {logging_mixin.py:190} INFO - Error processing batch 64: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=631 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146f640>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.490+0000] {logging_mixin.py:190} INFO - Processing batch 65: pages 641 to 650
[2025-03-28T05:37:11.492+0000] {logging_mixin.py:190} INFO - Scraping page 641...
[2025-03-28T05:37:11.498+0000] {logging_mixin.py:190} INFO - Error processing batch 65: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=641 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14527a0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.513+0000] {logging_mixin.py:190} INFO - Processing batch 66: pages 651 to 660
[2025-03-28T05:37:11.514+0000] {logging_mixin.py:190} INFO - Scraping page 651...
[2025-03-28T05:37:11.519+0000] {logging_mixin.py:190} INFO - Error processing batch 66: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=651 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14531c0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.533+0000] {logging_mixin.py:190} INFO - Processing batch 67: pages 661 to 670
[2025-03-28T05:37:11.535+0000] {logging_mixin.py:190} INFO - Scraping page 661...
[2025-03-28T05:37:11.540+0000] {logging_mixin.py:190} INFO - Error processing batch 67: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=661 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452ec0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.555+0000] {logging_mixin.py:190} INFO - Processing batch 68: pages 671 to 680
[2025-03-28T05:37:11.557+0000] {logging_mixin.py:190} INFO - Scraping page 671...
[2025-03-28T05:37:11.562+0000] {logging_mixin.py:190} INFO - Error processing batch 68: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=671 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146d5d0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.578+0000] {logging_mixin.py:190} INFO - Processing batch 69: pages 681 to 690
[2025-03-28T05:37:11.579+0000] {logging_mixin.py:190} INFO - Scraping page 681...
[2025-03-28T05:37:11.585+0000] {logging_mixin.py:190} INFO - Error processing batch 69: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=681 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452d40>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.599+0000] {logging_mixin.py:190} INFO - Processing batch 70: pages 691 to 700
[2025-03-28T05:37:11.601+0000] {logging_mixin.py:190} INFO - Scraping page 691...
[2025-03-28T05:37:11.605+0000] {logging_mixin.py:190} INFO - Error processing batch 70: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=691 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452e00>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.620+0000] {logging_mixin.py:190} INFO - Processing batch 71: pages 701 to 710
[2025-03-28T05:37:11.621+0000] {logging_mixin.py:190} INFO - Scraping page 701...
[2025-03-28T05:37:11.628+0000] {logging_mixin.py:190} INFO - Error processing batch 71: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=701 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14502b0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.646+0000] {logging_mixin.py:190} INFO - Processing batch 72: pages 711 to 720
[2025-03-28T05:37:11.647+0000] {logging_mixin.py:190} INFO - Scraping page 711...
[2025-03-28T05:37:11.652+0000] {logging_mixin.py:190} INFO - Error processing batch 72: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=711 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146d210>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.668+0000] {logging_mixin.py:190} INFO - Processing batch 73: pages 721 to 730
[2025-03-28T05:37:11.669+0000] {logging_mixin.py:190} INFO - Scraping page 721...
[2025-03-28T05:37:11.675+0000] {logging_mixin.py:190} INFO - Error processing batch 73: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=721 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14502b0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.690+0000] {logging_mixin.py:190} INFO - Processing batch 74: pages 731 to 740
[2025-03-28T05:37:11.692+0000] {logging_mixin.py:190} INFO - Scraping page 731...
[2025-03-28T05:37:11.698+0000] {logging_mixin.py:190} INFO - Error processing batch 74: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=731 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450250>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.713+0000] {logging_mixin.py:190} INFO - Processing batch 75: pages 741 to 750
[2025-03-28T05:37:11.715+0000] {logging_mixin.py:190} INFO - Scraping page 741...
[2025-03-28T05:37:11.720+0000] {logging_mixin.py:190} INFO - Error processing batch 75: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=741 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450790>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.734+0000] {logging_mixin.py:190} INFO - Processing batch 76: pages 751 to 760
[2025-03-28T05:37:11.735+0000] {logging_mixin.py:190} INFO - Scraping page 751...
[2025-03-28T05:37:11.740+0000] {logging_mixin.py:190} INFO - Error processing batch 76: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=751 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146dcc0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.754+0000] {logging_mixin.py:190} INFO - Processing batch 77: pages 761 to 770
[2025-03-28T05:37:11.755+0000] {logging_mixin.py:190} INFO - Scraping page 761...
[2025-03-28T05:37:11.760+0000] {logging_mixin.py:190} INFO - Error processing batch 77: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=761 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451fc0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.772+0000] {logging_mixin.py:190} INFO - Processing batch 78: pages 771 to 780
[2025-03-28T05:37:11.774+0000] {logging_mixin.py:190} INFO - Scraping page 771...
[2025-03-28T05:37:11.779+0000] {logging_mixin.py:190} INFO - Error processing batch 78: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=771 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451a50>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.793+0000] {logging_mixin.py:190} INFO - Processing batch 79: pages 781 to 790
[2025-03-28T05:37:11.794+0000] {logging_mixin.py:190} INFO - Scraping page 781...
[2025-03-28T05:37:11.799+0000] {logging_mixin.py:190} INFO - Error processing batch 79: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=781 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452050>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.813+0000] {logging_mixin.py:190} INFO - Processing batch 80: pages 791 to 800
[2025-03-28T05:37:11.814+0000] {logging_mixin.py:190} INFO - Scraping page 791...
[2025-03-28T05:37:11.819+0000] {logging_mixin.py:190} INFO - Error processing batch 80: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=791 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146efe0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.835+0000] {logging_mixin.py:190} INFO - Processing batch 81: pages 801 to 810
[2025-03-28T05:37:11.836+0000] {logging_mixin.py:190} INFO - Scraping page 801...
[2025-03-28T05:37:11.840+0000] {logging_mixin.py:190} INFO - Error processing batch 81: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=801 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1453790>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.854+0000] {logging_mixin.py:190} INFO - Processing batch 82: pages 811 to 820
[2025-03-28T05:37:11.855+0000] {logging_mixin.py:190} INFO - Scraping page 811...
[2025-03-28T05:37:11.860+0000] {logging_mixin.py:190} INFO - Error processing batch 82: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=811 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450d30>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.874+0000] {logging_mixin.py:190} INFO - Processing batch 83: pages 821 to 830
[2025-03-28T05:37:11.875+0000] {logging_mixin.py:190} INFO - Scraping page 821...
[2025-03-28T05:37:11.880+0000] {logging_mixin.py:190} INFO - Error processing batch 83: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=821 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14523e0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.894+0000] {logging_mixin.py:190} INFO - Processing batch 84: pages 831 to 840
[2025-03-28T05:37:11.895+0000] {logging_mixin.py:190} INFO - Scraping page 831...
[2025-03-28T05:37:11.900+0000] {logging_mixin.py:190} INFO - Error processing batch 84: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=831 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146f730>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.914+0000] {logging_mixin.py:190} INFO - Processing batch 85: pages 841 to 850
[2025-03-28T05:37:11.914+0000] {logging_mixin.py:190} INFO - Scraping page 841...
[2025-03-28T05:37:11.919+0000] {logging_mixin.py:190} INFO - Error processing batch 85: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=841 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452dd0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.932+0000] {logging_mixin.py:190} INFO - Processing batch 86: pages 851 to 860
[2025-03-28T05:37:11.933+0000] {logging_mixin.py:190} INFO - Scraping page 851...
[2025-03-28T05:37:11.938+0000] {logging_mixin.py:190} INFO - Error processing batch 86: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=851 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452680>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.951+0000] {logging_mixin.py:190} INFO - Processing batch 87: pages 861 to 870
[2025-03-28T05:37:11.952+0000] {logging_mixin.py:190} INFO - Scraping page 861...
[2025-03-28T05:37:11.956+0000] {logging_mixin.py:190} INFO - Error processing batch 87: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=861 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1453460>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.970+0000] {logging_mixin.py:190} INFO - Processing batch 88: pages 871 to 880
[2025-03-28T05:37:11.971+0000] {logging_mixin.py:190} INFO - Scraping page 871...
[2025-03-28T05:37:11.977+0000] {logging_mixin.py:190} INFO - Error processing batch 88: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=871 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146e6b0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:11.991+0000] {logging_mixin.py:190} INFO - Processing batch 89: pages 881 to 890
[2025-03-28T05:37:11.993+0000] {logging_mixin.py:190} INFO - Scraping page 881...
[2025-03-28T05:37:11.997+0000] {logging_mixin.py:190} INFO - Error processing batch 89: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=881 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452230>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.011+0000] {logging_mixin.py:190} INFO - Processing batch 90: pages 891 to 900
[2025-03-28T05:37:12.012+0000] {logging_mixin.py:190} INFO - Scraping page 891...
[2025-03-28T05:37:12.017+0000] {logging_mixin.py:190} INFO - Error processing batch 90: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=891 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14511e0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.032+0000] {logging_mixin.py:190} INFO - Processing batch 91: pages 901 to 910
[2025-03-28T05:37:12.034+0000] {logging_mixin.py:190} INFO - Scraping page 901...
[2025-03-28T05:37:12.039+0000] {logging_mixin.py:190} INFO - Error processing batch 91: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=901 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451e40>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.054+0000] {logging_mixin.py:190} INFO - Processing batch 92: pages 911 to 920
[2025-03-28T05:37:12.056+0000] {logging_mixin.py:190} INFO - Scraping page 911...
[2025-03-28T05:37:12.061+0000] {logging_mixin.py:190} INFO - Error processing batch 92: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=911 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14500d0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.077+0000] {logging_mixin.py:190} INFO - Processing batch 93: pages 921 to 930
[2025-03-28T05:37:12.078+0000] {logging_mixin.py:190} INFO - Scraping page 921...
[2025-03-28T05:37:12.084+0000] {logging_mixin.py:190} INFO - Error processing batch 93: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=921 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14535b0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.099+0000] {logging_mixin.py:190} INFO - Processing batch 94: pages 931 to 940
[2025-03-28T05:37:12.100+0000] {logging_mixin.py:190} INFO - Scraping page 931...
[2025-03-28T05:37:12.105+0000] {logging_mixin.py:190} INFO - Error processing batch 94: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=931 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452590>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.119+0000] {logging_mixin.py:190} INFO - Processing batch 95: pages 941 to 950
[2025-03-28T05:37:12.121+0000] {logging_mixin.py:190} INFO - Scraping page 941...
[2025-03-28T05:37:12.127+0000] {logging_mixin.py:190} INFO - Error processing batch 95: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=941 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146d990>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.141+0000] {logging_mixin.py:190} INFO - Processing batch 96: pages 951 to 960
[2025-03-28T05:37:12.142+0000] {logging_mixin.py:190} INFO - Scraping page 951...
[2025-03-28T05:37:12.195+0000] {logging_mixin.py:190} INFO - Error processing batch 96: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=951 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146e1d0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.226+0000] {logging_mixin.py:190} INFO - Processing batch 97: pages 961 to 970
[2025-03-28T05:37:12.235+0000] {logging_mixin.py:190} INFO - Scraping page 961...
[2025-03-28T05:37:12.260+0000] {logging_mixin.py:190} INFO - Error processing batch 97: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=961 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451b10>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.320+0000] {logging_mixin.py:190} INFO - Processing batch 98: pages 971 to 980
[2025-03-28T05:37:12.325+0000] {logging_mixin.py:190} INFO - Scraping page 971...
[2025-03-28T05:37:12.349+0000] {logging_mixin.py:190} INFO - Error processing batch 98: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=971 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450ca0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.387+0000] {logging_mixin.py:190} INFO - Processing batch 99: pages 981 to 990
[2025-03-28T05:37:12.392+0000] {logging_mixin.py:190} INFO - Scraping page 981...
[2025-03-28T05:37:12.420+0000] {logging_mixin.py:190} INFO - Error processing batch 99: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=981 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14501c0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.471+0000] {logging_mixin.py:190} INFO - Processing batch 100: pages 991 to 1000
[2025-03-28T05:37:12.477+0000] {logging_mixin.py:190} INFO - Scraping page 991...
[2025-03-28T05:37:12.487+0000] {logging_mixin.py:190} INFO - Error processing batch 100: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=991 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451b70>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.531+0000] {logging_mixin.py:190} INFO - Processing batch 101: pages 1001 to 1010
[2025-03-28T05:37:12.532+0000] {logging_mixin.py:190} INFO - Scraping page 1001...
[2025-03-28T05:37:12.539+0000] {logging_mixin.py:190} INFO - Error processing batch 101: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1001 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451bd0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.565+0000] {logging_mixin.py:190} INFO - Processing batch 102: pages 1011 to 1020
[2025-03-28T05:37:12.567+0000] {logging_mixin.py:190} INFO - Scraping page 1011...
[2025-03-28T05:37:12.578+0000] {logging_mixin.py:190} INFO - Error processing batch 102: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1011 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450130>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.601+0000] {logging_mixin.py:190} INFO - Processing batch 103: pages 1021 to 1030
[2025-03-28T05:37:12.607+0000] {logging_mixin.py:190} INFO - Scraping page 1021...
[2025-03-28T05:37:12.619+0000] {logging_mixin.py:190} INFO - Error processing batch 103: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1021 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146d810>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.646+0000] {logging_mixin.py:190} INFO - Processing batch 104: pages 1031 to 1040
[2025-03-28T05:37:12.648+0000] {logging_mixin.py:190} INFO - Scraping page 1031...
[2025-03-28T05:37:12.665+0000] {logging_mixin.py:190} INFO - Error processing batch 104: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1031 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146d090>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.687+0000] {logging_mixin.py:190} INFO - Processing batch 105: pages 1041 to 1050
[2025-03-28T05:37:12.692+0000] {logging_mixin.py:190} INFO - Scraping page 1041...
[2025-03-28T05:37:12.699+0000] {logging_mixin.py:190} INFO - Error processing batch 105: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1041 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451a50>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.728+0000] {logging_mixin.py:190} INFO - Processing batch 106: pages 1051 to 1060
[2025-03-28T05:37:12.729+0000] {logging_mixin.py:190} INFO - Scraping page 1051...
[2025-03-28T05:37:12.750+0000] {logging_mixin.py:190} INFO - Error processing batch 106: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1051 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450370>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.783+0000] {logging_mixin.py:190} INFO - Processing batch 107: pages 1061 to 1070
[2025-03-28T05:37:12.785+0000] {logging_mixin.py:190} INFO - Scraping page 1061...
[2025-03-28T05:37:12.793+0000] {logging_mixin.py:190} INFO - Error processing batch 107: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1061 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452bc0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.820+0000] {logging_mixin.py:190} INFO - Processing batch 108: pages 1071 to 1080
[2025-03-28T05:37:12.822+0000] {logging_mixin.py:190} INFO - Scraping page 1071...
[2025-03-28T05:37:12.830+0000] {logging_mixin.py:190} INFO - Error processing batch 108: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1071 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450940>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.849+0000] {logging_mixin.py:190} INFO - Processing batch 109: pages 1081 to 1090
[2025-03-28T05:37:12.851+0000] {logging_mixin.py:190} INFO - Scraping page 1081...
[2025-03-28T05:37:12.858+0000] {logging_mixin.py:190} INFO - Error processing batch 109: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1081 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450520>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.876+0000] {logging_mixin.py:190} INFO - Processing batch 110: pages 1091 to 1100
[2025-03-28T05:37:12.878+0000] {logging_mixin.py:190} INFO - Scraping page 1091...
[2025-03-28T05:37:12.883+0000] {logging_mixin.py:190} INFO - Error processing batch 110: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1091 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1453520>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.903+0000] {logging_mixin.py:190} INFO - Processing batch 111: pages 1101 to 1110
[2025-03-28T05:37:12.905+0000] {logging_mixin.py:190} INFO - Scraping page 1101...
[2025-03-28T05:37:12.912+0000] {logging_mixin.py:190} INFO - Error processing batch 111: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1101 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146dae0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.932+0000] {logging_mixin.py:190} INFO - Processing batch 112: pages 1111 to 1120
[2025-03-28T05:37:12.934+0000] {logging_mixin.py:190} INFO - Scraping page 1111...
[2025-03-28T05:37:12.941+0000] {logging_mixin.py:190} INFO - Error processing batch 112: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1111 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146f640>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.962+0000] {logging_mixin.py:190} INFO - Processing batch 113: pages 1121 to 1130
[2025-03-28T05:37:12.964+0000] {logging_mixin.py:190} INFO - Scraping page 1121...
[2025-03-28T05:37:12.970+0000] {logging_mixin.py:190} INFO - Error processing batch 113: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1121 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451cc0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:12.993+0000] {logging_mixin.py:190} INFO - Processing batch 114: pages 1131 to 1140
[2025-03-28T05:37:12.995+0000] {logging_mixin.py:190} INFO - Scraping page 1131...
[2025-03-28T05:37:13.004+0000] {logging_mixin.py:190} INFO - Error processing batch 114: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1131 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452b90>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.023+0000] {logging_mixin.py:190} INFO - Processing batch 115: pages 1141 to 1150
[2025-03-28T05:37:13.025+0000] {logging_mixin.py:190} INFO - Scraping page 1141...
[2025-03-28T05:37:13.031+0000] {logging_mixin.py:190} INFO - Error processing batch 115: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1141 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450dc0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.046+0000] {logging_mixin.py:190} INFO - Processing batch 116: pages 1151 to 1160
[2025-03-28T05:37:13.047+0000] {logging_mixin.py:190} INFO - Scraping page 1151...
[2025-03-28T05:37:13.051+0000] {logging_mixin.py:190} INFO - Error processing batch 116: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1151 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1450220>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.068+0000] {logging_mixin.py:190} INFO - Processing batch 117: pages 1161 to 1170
[2025-03-28T05:37:13.069+0000] {logging_mixin.py:190} INFO - Scraping page 1161...
[2025-03-28T05:37:13.075+0000] {logging_mixin.py:190} INFO - Error processing batch 117: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1161 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452ec0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.089+0000] {logging_mixin.py:190} INFO - Processing batch 118: pages 1171 to 1180
[2025-03-28T05:37:13.091+0000] {logging_mixin.py:190} INFO - Scraping page 1171...
[2025-03-28T05:37:13.097+0000] {logging_mixin.py:190} INFO - Error processing batch 118: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1171 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146ffa0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.108+0000] {logging_mixin.py:190} INFO - Processing batch 119: pages 1181 to 1190
[2025-03-28T05:37:13.109+0000] {logging_mixin.py:190} INFO - Scraping page 1181...
[2025-03-28T05:37:13.114+0000] {logging_mixin.py:190} INFO - Error processing batch 119: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1181 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e157b130>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.125+0000] {logging_mixin.py:190} INFO - Processing batch 120: pages 1191 to 1200
[2025-03-28T05:37:13.126+0000] {logging_mixin.py:190} INFO - Scraping page 1191...
[2025-03-28T05:37:13.130+0000] {logging_mixin.py:190} INFO - Error processing batch 120: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1191 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146d210>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.141+0000] {logging_mixin.py:190} INFO - Processing batch 121: pages 1201 to 1210
[2025-03-28T05:37:13.142+0000] {logging_mixin.py:190} INFO - Scraping page 1201...
[2025-03-28T05:37:13.146+0000] {logging_mixin.py:190} INFO - Error processing batch 121: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1201 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14530a0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.158+0000] {logging_mixin.py:190} INFO - Processing batch 122: pages 1211 to 1220
[2025-03-28T05:37:13.159+0000] {logging_mixin.py:190} INFO - Scraping page 1211...
[2025-03-28T05:37:13.163+0000] {logging_mixin.py:190} INFO - Error processing batch 122: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1211 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e14512a0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.176+0000] {logging_mixin.py:190} INFO - Processing batch 123: pages 1221 to 1230
[2025-03-28T05:37:13.178+0000] {logging_mixin.py:190} INFO - Scraping page 1221...
[2025-03-28T05:37:13.183+0000] {logging_mixin.py:190} INFO - Error processing batch 123: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1221 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e157bf10>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.198+0000] {logging_mixin.py:190} INFO - Processing batch 124: pages 1231 to 1240
[2025-03-28T05:37:13.199+0000] {logging_mixin.py:190} INFO - Scraping page 1231...
[2025-03-28T05:37:13.204+0000] {logging_mixin.py:190} INFO - Error processing batch 124: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1231 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451990>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.218+0000] {logging_mixin.py:190} INFO - Processing batch 125: pages 1241 to 1250
[2025-03-28T05:37:13.219+0000] {logging_mixin.py:190} INFO - Scraping page 1241...
[2025-03-28T05:37:13.224+0000] {logging_mixin.py:190} INFO - Error processing batch 125: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1241 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1452a10>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.238+0000] {logging_mixin.py:190} INFO - Processing batch 126: pages 1251 to 1260
[2025-03-28T05:37:13.239+0000] {logging_mixin.py:190} INFO - Scraping page 1251...
[2025-03-28T05:37:13.244+0000] {logging_mixin.py:190} INFO - Error processing batch 126: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1251 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1451030>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.258+0000] {logging_mixin.py:190} INFO - Processing batch 127: pages 1261 to 1270
[2025-03-28T05:37:13.259+0000] {logging_mixin.py:190} INFO - Scraping page 1261...
[2025-03-28T05:37:13.265+0000] {logging_mixin.py:190} INFO - Error processing batch 127: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1261 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146e410>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.280+0000] {logging_mixin.py:190} INFO - Processing batch 128: pages 1271 to 1280
[2025-03-28T05:37:13.281+0000] {logging_mixin.py:190} INFO - Scraping page 1271...
[2025-03-28T05:37:13.287+0000] {logging_mixin.py:190} INFO - Error processing batch 128: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1271 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e146cf40>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.300+0000] {logging_mixin.py:190} INFO - Processing batch 129: pages 1281 to 1290
[2025-03-28T05:37:13.302+0000] {logging_mixin.py:190} INFO - Scraping page 1281...
[2025-03-28T05:37:13.307+0000] {logging_mixin.py:190} INFO - Error processing batch 129: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1281 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1453550>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.321+0000] {logging_mixin.py:190} INFO - Processing batch 130: pages 1291 to 1300
[2025-03-28T05:37:13.322+0000] {logging_mixin.py:190} INFO - Scraping page 1291...
[2025-03-28T05:37:13.328+0000] {logging_mixin.py:190} INFO - Error processing batch 130: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1291 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e1453490>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.341+0000] {logging_mixin.py:190} INFO - Processing batch 131: pages 1301 to 1310
[2025-03-28T05:37:13.342+0000] {logging_mixin.py:190} INFO - Scraping page 1301...
[2025-03-28T05:37:13.348+0000] {logging_mixin.py:190} INFO - Error processing batch 131: HTTPSConnectionPool(host='default.url', port=443): Max retries exceeded with url: /project-swasta?page=1301 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f25e157bbb0>: Failed to resolve 'default.url' ([Errno -2] Name or service not known)"))
[2025-03-28T05:37:13.362+0000] {logging_mixin.py:190} INFO - Scraping complete. Total rows scraped: 0
[2025-03-28T05:37:13.363+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-28T05:37:13.373+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-28T05:37:13.375+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=split_web_scraping_v01, task_id=scrape_and_store, run_id=manual__2025-03-28T05:37:07.809707+00:00, execution_date=20250328T053707, start_date=20250328T053709, end_date=20250328T053713
[2025-03-28T05:37:13.431+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-28T05:37:13.453+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-28T05:37:13.457+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
